sdks/python/apache_beam/yaml/main.py

```diff
@@ -51,19 +51,22 @@ def _pipeline_spec_from_args(known_args):
     raise ValueError(
         "Exactly one of pipeline_spec or pipeline_spec_file must be set.")
 
-  return yaml.load(pipeline_yaml, Loader=yaml_transform.SafeLineLoader)
+  return pipeline_yaml
 
 
 def run(argv=None):
   yaml_transform._LOGGER.setLevel('INFO')
   known_args, pipeline_args = _configure_parser(argv)
-  pipeline_spec = _pipeline_spec_from_args(known_args)
+  pipeline_yaml = _pipeline_spec_from_args(known_args)
+  pipeline_spec = yaml.load(pipeline_yaml, Loader=yaml_transform.SafeLineLoader)
 
-  with beam.Pipeline(options=beam.options.pipeline_options.PipelineOptions(
-      pipeline_args,
-      pickle_library='cloudpickle',
-      **yaml_transform.SafeLineLoader.strip_metadata(pipeline_spec.get(
-          'options', {})))) as p:
+  with beam.Pipeline(  #
```

Comment:
erroneous '#'

Reply:
This is actually to prevent some odd yapf formatting. Clarifying.